tests/conftest.py

```diff
@@ -312,6 +312,18 @@ def messages_successful_response() -> Dict[str, Any]:
     })
 
 
+@pytest.fixture(params=['', '1'])
+def zulip_feature_level(request):
```

Comment:
While `zulip_feature_level` isn't used anywhere in this particular commit, this appeared to be a good place to introduce the fixture as this is where we start to fetch the server version and the feature level.